docs/reference/msbuild-targets.md

```diff
@@ -221,7 +221,7 @@ When using `MSBuild -t:pack -p:IsTool=true`, all output files, as specified in t
 
 ### Packing using a .nuspec
 
-You can use a `.nuspec` file to pack your project provided that you have a SDK project file to import `NuGet.Build.Tasks.Pack.targets` so that the pack task can be executed. You still need to restore the project before you can pack a nuspec file. The target framework of the project file is irrelevant and not used when packing a nuspec. The following three MSBuild properties are relevant to packing using a `.nuspec`:
+Although it is recommended that you [include all the properties](../reference/msbuild-targets.md#pack-target) that are usually in the `.nuspec` file in the project file instead, you can choose to use a `.nuspec` file to pack your project. To use `.nuspec`, you must have an SDK project file that imports `NuGet.Build.Tasks.Pack.targets` so that the pack task can be executed. You still need to restore the project before you can pack a nuspec file. The target framework of the project file is irrelevant and not used when packing a nuspec. The following three MSBuild properties are relevant to packing using a `.nuspec`:
```

Comment:
> To use `.nuspec`, you must have an SDK project file that imports `NuGet.Build.Tasks.Pack.targets` so that the pack task can be executed

I think this is out of date. 

SDK projects include the pack targets by default. 

non-SDK based PackageReference projects don't